extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1238,11 +1238,13 @@ export class AmpA4A extends AMP.BaseElement {
   /**
    * Gets the Ad URL to send an XHR Request to.  To be implemented
    * by network.
-   * @param {?CONSENT_POLICY_STATE} unusedConsentState
+   * @param {?object} unusedConsentTuple
```

Comment:
Is this ever nullable given you appear to always be passing an explicit object?

Reply:
I keep all params nullable because some getAdUrl() call sites don't pass in anything. Like the one in [amp-ad-network-adzerk-impl.js](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-network-adzerk-impl/0.1/amp-ad-network-adzerk-impl.js#L91)